tutorials/hello-blockstack.md

```diff
@@ -115,13 +114,11 @@ With the first condition (when the user is signed in), we load the user data fro
 
 ```js
 if (blockstack.isUserSignedIn()) {
-  blockstack.loadUserData(function(userData) {
-    showProfile(userData.profile)
-  })
+  showProfile(blockstack.loadUserData())
 } else if (blockstack.isSignInPending()) {
-  blockstack.signUserIn(function(userData) {
-    window.location = window.location.origin
-  })
+  blockstack.handlePendingSignIn().then(function(userData) {
+      window.location = window.location.origin
+    });
```

Comment:
this semicolon probably shouldn't be here for the sake of consistency.
other than that, looks good to me